hyperjumptech/monika
# Added: Prometheus Alert Metric

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Resolves issue #612  

## How did you implement / how did you fix it  
1.  Add a new Prometheus metric named `monika_alert_total`
2. Collect the metric when an alert triggered

## How to test  
1. Run HTTP server target
3. Run Prometheus server
4. Run Monika and probe the HTTP server target
5. Stop HTTP server target and let the alert triggered
6. Run Prometheus query. For instance `rate(monika_alert_total[1m])` and see the graph goes up
7. Run the HTTP server target again and see the graph goes down

## Video
https://user-images.githubusercontent.com/15191978/177974673-ae0b9413-edcc-4049-9344-06ac8f5176c9.mp4
